signer/src/stacks/contracts.rs

```diff
@@ -1109,6 +1114,9 @@ impl AsContractCall for RejectWithdrawalV1 {
     /// 6. Whether the withdrawal request has expired. Fail if it hasn't.
     /// 7. Whether the withdrawal request is being serviced by a sweep
     ///    transaction that is in the mempool.
+    /// 8. Whether we need to worry about forks causing the withdrawal to
+    ///    be confirmed by a sweep that was broadcst changing the status of
```

Comment:
```suggestion
    ///    be confirmed by a sweep that was broadcast changing the status of
```

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1411/commits/d54581faabf3dfb0e5acdd15056a6e4654eb25cb